### PR TITLE
Bug #307 Fix: Notifications stopped working

### PR DESCRIPTION
**Pull request by alexanderlz**
Updated to use the newer Notification object (as described in MDN)

### CODE REVIEW

**Comment by joeysim** [comment]
Where's the like button?

File: rd_ui/app/scripts/services/notifications.js [hunk 1]
```diff
@@ -27,23 +30,13 @@
                 return;
             }
 
-            if (document.webkitVisibilityState && document.webkitVisibilityState == 'visible') {
```
Comment (arikfr) [inline rd_ui/app/scripts/services/notifications.js]: This is here to make sure we don't show notifications when the page is in focus. Is there a reason you removed it?

File: rd_ui/app/scripts/services/notifications.js [hunk 2]
```diff
@@ -27,23 +30,13 @@
...
-            }
-
-            if (lastNotification) {
-                lastNotification.cancel();
-            }
-
-            var notification = window.webkitNotifications.createNotification('', title, content);
-            lastNotification = notification;
+            //using the 'tag' to avoid showing duplicate notifications
+            var notification = new Notification(title, {'tag': title+content, 'body': content});
```
Comment (arikfr) [inline rd_ui/app/scripts/services/notifications.js]: Why aren't you using the title anymore?

File: rd_ui/app/scripts/services/notifications.js (hunk 1 above)
Comment (alexanderlz) [inline rd_ui/app/scripts/services/notifications.js]: To emphasize the effect of notifying!
the truth is I removed it while testing to always see the notifications, and decided it's not that bad to keep it that way.
wdyt?

File: rd_ui/app/scripts/services/notifications.js (hunk 2 above)
Comment (alexanderlz) [inline rd_ui/app/scripts/services/notifications.js]: We are.
the title is the first parameter passed to the constructor

Comment (arikfr) [inline rd_ui/app/scripts/services/notifications.js]: :facepalm:

**Pull request merged**